src/lib/isLength.js

```diff
@@ -5,15 +5,42 @@ export default function isLength(str, options) {
   assertString(str);
   let min;
   let max;
+  let exact;
+  let result;
+  let isPerfect = false;
   if (typeof (options) === 'object') {
     min = options.min || 0;
     max = options.max;
+    exact = options.exact;
   } else { // backwards compatibility: isLength(str, min [, max])
     min = arguments[1] || 0;
     max = arguments[2];
+    exact = arguments[3];
```

Comment:
I don't think we need backwards compatibility for a new feature

Reply:
That also removes the need for some of the tests btw

Reply:
Ohh I thought if at all a User wants to check if it's exact to a bunch of values rather than a single value, so I configured the exact value to be an array or an object or a number. Should I remove the backwards compatibility feature ??

Reply:
It's just this line that you can remove (and the tests associated to this line). This else statement is only here for the people that use the old syntax for isLength. And since this is a new feature there is no need to implement it in both the old and new syntax.

That you configured the exact value to be an array or object or number is fine.

Reply:
Okkay!! I made the required changes & also updated the Tests for the same... Hence, now this new Feature exact is implemented only in the new Syntax of isLength.